CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Change Log
 
-## v0.x.x Unreleased
+## v0.15.0 Released
```

Comment:
This should be the date it was released on

Reply:
Will set that when I release tommorrow!

Reply:
On Friday*

Reply:
Perfect, thanks

Reply:
In case it helps, we also have https://python.arviz.org/en/latest/contributing/how_to_release.html